src/main/java/uk/gov/hmcts/befta/dse/ccd/definition/converter/JsonTransformer.java

```diff
@@ -35,7 +35,7 @@
             "AuthorisationCaseField", "AuthorisationCaseState", "AuthorisationCaseType", "AuthorisationComplexType",
             "CaseEventToFields", "CaseField", "CaseRoles", "CaseTypeTab" ,"SearchInputFields", "SearchResultFields", "State",
             "WorkBasketInputFields", "WorkBasketResultFields", "Category", "Banner", "CaseType", "ComplexTypes", "EventToComplexTypes",
-            "FixedLists", "Jurisdiction", "UserProfile","SearchAlias", "SearchCasesResultFields");
+            "FixedLists", "Jurisdiction", "UserProfile","SearchAlias", "SearchCasesResultFields", "NoCConfig");
```

Comment:
Hi Kiran. Can we make the name of the tab more descriptive? i.e NoticeOfChangeConfig, instead of NoCConfig.

Reply:
chanded NoCConfig tab to NoticeOfChangeConfig tab